content/en/docs/reference/instrumentation/kubelet-summary-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Kubelet Summary API
+content_type: reference
+weight: 55
+description: >-
+  The kubelet Summary API for accessing node, pod, and container metrics.
+---
+
+<!-- overview -->
+
+The kubelet exposes a Summary API at the `/stats/summary` endpoint that provides
+aggregated metrics for nodes, pods, containers, and volumes. This API serves as a
+primary source for resource usage data collected by the kubelet.
+
+<!-- body -->
+
+## Accessing the Summary API
+
+You can access the Summary API through the Kubernetes API server as a proxy:
+
+```shell
+kubectl get --raw "/api/v1/nodes/<NODE_NAME>/proxy/stats/summary"
+```
+
+Replace `<NODE_NAME>` with the name of the node you want to query.
+
+Alternatively, if you have direct access to the kubelet port (typically 10250),
+you can query it directly:
+
+```shell
+curl -k https://<NODE_IP>:10250/stats/summary
+```
+
+{{< note >}}
+Direct access to the kubelet requires proper authentication and is typically
+restricted in production environments. The `-k` flag bypasses certificate 
+verification, leaving the connection subject to MITM attacks. Using `kubectl 
+proxy` is the recommended approach for secure access.
+{{< /note >}}
+
+## API Response Structure
+
+The Summary API returns a JSON object containing comprehensive metrics organized
+hierarchically:
+
+```json
+{
+  "node": {
+    "nodeName": "node-1",
+    "systemContainers": [...],
+    "startTime": "2024-01-01T00:00:00Z",
+    "cpu": {...},
+    "memory": {...},
+    "network": {...},
+    "fs": {...},
+    "runtime": {...},
+    "rlimit": {...}
+  },
+  "pods": [
+    {
+      "podRef": {
+        "name": "pod-name",
+        "namespace": "namespace",
+        "uid": "pod-uid"
+      },
+      "startTime": "2024-01-01T00:00:00Z",
+      "containers": [...],
+      "cpu": {...},
+      "memory": {...},
+      "network": {...},
+      "volume": [...],
+      "ephemeral-storage": {...},
+      "process_stats": {...}
+    }
+  ]
+}
+```
+
+## Metrics Categories
+
+### Node-Level Metrics
```

Comment:
are these section breakdowns too much detail - should we just mention at a high level across all scopes (Node/Pod/Container)?